build/make_bundle.sh

```diff
@@ -145,3 +146,21 @@ yq w $OLM_CATALOG/splunk/splunk.package.yaml packageName "splunk-certified" > $O
 
 # Mac OS expects sed -i '', Linux expects sed -i''. To workaround this, using .bak
 zip $OLM_CERTIFIED/splunk.zip -j $OLM_CERTIFIED/splunk $OLM_CERTIFIED/splunk/*
+
+# This adds the 'protocol' field back to the CRDs.
+function updateCRDS {
+for crd in `ls $1`
+    do
+        echo Updating crd: $crd
+        line_num=`grep -n "x-kubernetes-list-map-keys" $1/$crd | awk -F ":" '{print$1}'`
+        line_num=$(($line_num-2))
+        awk 'NR==v1{print "                          - protocol"}1' v1="${line_num}" $1/$crd > tmp.out
+        mv tmp.out $1/$crd
+    done
+}
+
+echo Updating $CRDS_PATH
+updateCRDS $CRDS_PATH
+
+echo Updating $OLM_CATALOG/splunk/$VERSION
+updateCRDS $OLM_CATALOG/splunk/$VERSION
```

Comment:
Nit: Can we put all the changes in one block, surrounded by a comment on why we are doing it, and which ideally it should go away in future Operator SDKs. So, that we don't lose track of it, on future SDK upgrades.